Wireframe/css/style.css

```diff
@@ -0,0 +1,97 @@
+
+body {
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: 100px;
+  margin-right: 100px;
+  margin-bottom: 0;
+  background: #f0efe7;
+  padding: 0;
+  height: 100%;
+}
+
+header{
+  text-align: center;
+  margin-bottom: 5px;
+ 
+}
+h1,h2{
+  margin:5px;
+ 
+}
+article{
+  padding-left: 20px;
```

Comment:
It will look better if you add padding not only to the left, but to all 4 sides. Let's try it, shall we?